src/main.ts

```diff
@@ -61,35 +74,46 @@ async function run(): Promise<void> {
 
     if (pullRequest) {
       const {
-        repo: {repo: repoName, owner: repoOwner}
+        repo: { repo: repoName, owner: repoOwner }
       } = github.context
       const defaultParameter = {
         repo: repoName,
         owner: repoOwner
       }
       // Find unique comments
-      const {data: comments} = await octokit.rest.issues.listComments({
+      const { data: comments } = await octokit.rest.issues.listComments({
         ...defaultParameter,
         issue_number: pullRequest.number
       })
-      const targetComment = comments.find(c => {
-        c?.body?.includes(IDENTIFIER)
-      })
+      const targetComments = comments.filter(c => c?.body?.includes(IDENTIFIER))
       // Delete previous comment if exist
-      if (targetComment) {
+      for (const comment of targetComments) {
         await octokit.rest.issues.deleteComment({
           ...defaultParameter,
-          comment_id: targetComment.id
+          comment_id: comment.id
         })
         core.info(
-          `Comment successfully delete for id: ${String(targetComment.id)}`
+          `Comment successfully delete for id: ${String(targetComments[0].id)}`
```

Comment:
Would `targetComments[0].id` be the same as `comment.id` for each item in `targetComments`?